pandas/tests/io/excel/test_xlrd.py

```diff
@@ -0,0 +1,344 @@
+import contextlib
+from functools import partial
+import os
+import warnings
+
+import numpy as np
+import pytest
+
+import pandas.util._test_decorators as td
+
+from pandas import DataFrame
+from pandas.core.config import get_option, set_option
+import pandas.util.testing as tm
+from pandas.util.testing import ensure_clean
+
+from pandas.io.excel import ExcelFile, read_excel
+
+_seriesd = tm.getSeriesData()
+_tsd = tm.getTimeSeriesData()
+_frame = DataFrame(_seriesd)[:10]
+_frame2 = DataFrame(_seriesd, columns=['D', 'C', 'B', 'A'])[:10]
+_tsframe = tm.makeTimeDataFrame()[:5]
+_mixed_frame = _frame.copy()
+_mixed_frame['foo'] = 'bar'
+
+
+@contextlib.contextmanager
+def ignore_xlrd_time_clock_warning():
```

Comment:
any code required by both `test_base.py` and `test_xlrd.py` should probably be in `test_base.py`